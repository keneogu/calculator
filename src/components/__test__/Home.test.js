import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../Home';

test('should have a greeting text', () => {
  const { getByTestId } = render(<Home />);
  const greetElement = getByTestId('h-text');

  expect(greetElement.textContent).toBe('Welcome to the Home Page');
});

test('matches the home snapshot', () => {
  const tree = renderer.create(<Home />).toJSON();

  expect(tree).toMatchSnapshot();
});
