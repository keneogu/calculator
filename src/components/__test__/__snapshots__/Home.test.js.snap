// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the home snapshot 1`] = `
<div
  className="homepage"
>
  <h1
    className="home-head"
    data-testid="h-text"
  >
    Welcome to the Home Page
  </h1>
  <p
    className="home-para"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
  </p>
</div>
`;
